src/Service/SymfonyVersionProvider.php

```diff
@@ -45,4 +45,32 @@ public function getCurrentVersion()
             return $version;
         });
     }
+
+    /**
+     * @throws \RuntimeException
+     */
+    public function getSupportedVersions(): array
+    {
+        $httpClient = $this->httpClient;
+
+        $versions = $this->cache->get('symfony_supported_versions', function (ItemInterface $item) use ($httpClient) {
+            try {
+                $response = $httpClient->request('GET', 'https://symfony.com/releases.json');
+                $data = $response->toArray(true);
+                $versions = $data['supported_versions'] ?? null;
+            } catch (\Throwable $e) {
+                $versions = null;
+            }
+
+            $item->expiresAfter(null === $versions ? 300 : 604800);
```

Comment:
What would happen the first 7 days after, for example, 5.3 is released? Actually, I don't know the exact moment that a new release is added to `releases.json`, so I'm wondering if - especially with the caching - if there will be some time when a version is supported and making PR to it IS valid... but it's not in the version yet. Also... I think the new branches (e.g. 5.3) is created a few weeks before the stable release.

Reply:
We could relax this a bit more. Ie, shorter cache. 

Two things can happen: 
1) A user submits a PR to a branch that is not supported and we dont make a comment.
2) A user submits a PR to a branch that is just created (5.3) and we do make a comment that it is not supported. 

Number 2 is an issue, but if that happens the first day.. then it is fine. I'll reduce this to 1 day